mypy/semanal.py

```diff
@@ -964,7 +965,9 @@ def analyze_base_classes(self, defn: ClassDef) -> None:
                     self.fail("Cannot subclass NewType", defn)
                 base_types.append(base)
             elif isinstance(base, AnyType):
-                if self.options.disallow_subclassing_any:
+                # if --disallow-implicit-any-types is set, the issue is reported later
+                implicit = self.options.disallow_implicit_any_types and base.is_from_silent_import
+                if self.options.disallow_subclassing_any and not implicit:
```

Comment:
Yeah, I think it's okay. As long as it's not overwhelming for the user, it's actually better to have more information than less. The new message would explain more. This is what it outputs now for a simple case:
```
error: Cannot find module named '
note: (Perhaps setting MYPYPATH or using the "--ignore-missing-imports" flag would help)
error: Class cannot subclass 'Unchecked' (has type 'Any')
error: Base type Unchecked is implicitly converted to "Any" due to import from unanalyzed module
```